src/lib.rs

```diff
@@ -2610,3 +2621,62 @@ impl SwapChain {
         }
     }
 }
+
+type ErrorSinkCallback = dyn Fn(GPUError) ;
+type ErrorSink = Arc<Mutex<ErrorSinkRaw>>;
+
+
+/// TODO:
+#[derive(Debug)]
+pub struct GPUError {
```

Comment:
we don't have any `GPU` prefixed types. Could this be `AnyError`?

Reply:
Ah yes, as I mentioned in the description, I couldn't call it `Error` because the std `Error` is imported, and was gonna rename that import later. But I was gonna call it just `Error`, for being consistent with the `GPUFoo` -> `wgpu::Foo` convention. 

It does kind of look like it should impl std `Error` as well for being more rusty, though it doesn't matter much as it is not used as a `Result::Err` in the public interface.